tests/baselines/reference/callbackTag1.types

```diff
@@ -20,7 +20,7 @@ var sid = s => s + "!";
 /** @type {NoReturn} */
 var noreturn = obj => void obj.title
 >noreturn : NoReturn
->obj => void obj.title : (obj: { e: number; m: number; title: string; }) => any
+>obj => void obj.title : (s: { e: number; m: number; title: string; }) => any

```

Comment:
These .types baselines are okay and expected, I think? We're now obeying the `@type` tag, so the output is, e.g. the `NoReturn` type, and the `NoReturn` parameter name is `s`.